wiki/Ranking/en.md

```diff
@@ -0,0 +1,44 @@
+---
+stub: true
+tags:
+  - pp
+  - leaderboards
+  - ranking
+---
+
+# Ranking
+
+**Ranking** (alternatively called "leaderboards") is a list that sorts members of the osu! community according to certain criteria. This can also refer to one's position in such a list.
+
+osu! currently has five types of rankings:
+
+- Performance points ranking: based on [performance points](/wiki/Performance_points).
+- Score ranking: based on total ranked score.
+- Kudosu! ranking: based on [kudosu!](/wiki/Modding/Kudosu) count.
+- Spotlights rankings: based on [score](/wiki/Score) on the currently spotlighted beatmaps.
+- Beatmap rankings: based on score for individual [beatmaps](/wiki/Beatmaps).
+- Country rankings: based on performance points of players from a country.
+
+The performance points ranking, score ranking, and Kudosu! ranking can be navigated to by using the `Rankings` dropdown panel at the top of the webpage and choosing the `Performance` option.
+
+## Performance points ranking
+
+*Main page: [Performance points](/wiki/Performance_points)*
+
+The performance points ranking describes a player's skill-level in comparison to other players. This leaderboard is determined by the total number of performance points an individual player has, and can be seen on a player's profile and on the [performance leaderboards](https://osu.ppy.sh/rankings/osu/performance).
+
+## Score ranking
+
+Score ranking compares players' total score and lists them accordingly. Ranking highly on this leaderboard requires a large time commitment, but requires relatively less mechanical skill than the performance points ranking.
+
+## Kudosu! ranking
```

Comment:
Hmm, I'm not even sure what is the official way to format this. See https://osu.ppy.sh/help/wiki/Modding/Kudosu and the profile for example. By the way, I'm leaning towards the `kudosu!` format.

```suggestion
## kudosu! ranking
```

Reply:
ok, I will note this as well